### DISCUSSION TITLE
Binaries for Rofi 1.7?

### DISCUSSION THREAD

**Discussion by alabnt** [davatorium/rofi#1515] [Q&A] [answered]
Title, anyone got precompiled binaries for rofi 1.7?
Thanks.

**Comment by alabnt** [comment, marked as answer]
Also apologies if I got the category wrong.

**Reply by DaveDavenport** [reply]
No, we do not ship binaries, snaps,appimages or other pre-compiled version of rofi.

Please check your distribution for a compiled version of rofi.

**Comment by alabnt** [comment]
ah ok. Thanks anyways :)

**Reply by eadmaster** [reply]
there is one in nix, but it is not working fine in my test machine.
